infrastructure/constructs/processing.py

```diff
@@ -69,6 +83,33 @@ def __init__(
             "AmazonS3ReadOnlyAccess"
         )
 
+        ############################################################################################
+        # ROOT CATALOG UPDATE MESSAGE QUEUE
+
+        self.message_queue = aws_sqs.Queue(
+            self, "root-catalog-message-queue", visibility_timeout=Duration.seconds(60)
```

Comment:
Since this is tied to the Lambda timeout we should probably pull out a constant for it.